synse/protocols/modbus/modbus_common.py

```diff
@@ -37,6 +54,31 @@ def get_fan_frequency_gs3(ser, slave_address):
     return read_holding_register(ser, slave_address, 0x0002)
 
 
+def get_fan_rpm_ecblue(ser, slave_address):
+    """Production only rpm reads from ECblue fan (vapor_fan).
+    :param ser: Serial connection to the ECblue fan controller.
+    :param slave_address: The Modbus slave address of the device.
+    :return: The fan speed in rpm.
+    """
+    # TODO: There may be another register for this.
+    # We are reading off of the Hz setting and there is going to be ramp up/down
+    # time which is not taken into account here.
+    max_rpm = get_fan_max_rpm_ecblue(ser, slave_address)
+    logger.debug('max_rpm: {}'.format(max_rpm))
+
+    # Get the hz setting in Hz * 10.
+    hz_setting = read_holding_register(ser, slave_address, 0x02)
+    logger.debug('raw Hz setting: 0x{:04x}'.format(hz_setting))
+    # Convert from hex to BCD.
+    hz_setting_string = '{:04x}'.format(hz_setting)
+    hz_setting_int = int('{}'.format(hz_setting_string))
```

Comment:
i believe this could be `int(hz_setting_string)` (no additional string format required)